user/troubleshooting/installation-troubleshooting.md

```diff
@@ -77,6 +77,15 @@ If installing the available drivers does not help, disable the network card in t
 If this solves the issue, it confirms the PCI card is incompatible with Qubes. 
 In this case, you may want to consider replacing it with a network card of a different brand. 
 Broadcom cards are notoriously problematic with Qubes.
- 
 
 
+## "Unsupported Hardware Detected" error ##
+
+During Qubes installation, you may come across the error message which reads "Unsupported Hardware Detected. 
+Missing features: IOMMU/VT-d/AMD-Vi, Interrupt Remapping. Without these features, Qubes OS will not function normally". 
+
+This error message indicates that IOMMU-virtualization hasn’t been activated in the BIOS. 
+Return to the [hardware requirements][hardware-reqs] section to learn how to activate it. 
+If the setting is not configured correctly, it means that your hardware won’t be able to leverage some Qubes security features, such as a strict isolation of the networking and USB hardware.
```

Comment:
This is getting more strict with recent versions:
- Qubes 4.0: default installation won't function properly without IOMMU, as default sys-net and sys-usb qubes require IOMMU; it's possible to configure then to reduce isolation and not use IOMMU by changing virtualization mode of those two VMs to "PV"
- Qubes 4.1: IOMMU is strictly required, even for PV; it is not possible to use Qubes on a system without IOMMU